app/assets/stylesheets/custom.scss

```diff
@@ -19,3 +19,59 @@
     margin-bottom: 0;
   }
 }
+
+.locale .locale-form label {
+  color: #000;
+}
+
+body:not(.admin) {
+  .top-links {
+    @include full-width-border(bottom, 1px solid $border, $adjust-padding: true);
+
+    background: #fdfdfd;
+    text-align: right;
+  }
+}
+
+.top-bar {
+  .top-bar-title a img {
+    width: 40mm;
+    margin: 20px;
+  }
+}
+
+.public > .wrapper > header {
+  .top-bar {
+    color: #000;
+    background-color: #fdfdfd;
```

Comment:
We can use `$black` and `$white` in order to be consistent with the rest of the web.